### ISSUE TITLE
github action 'Build icons' not working as intended

### ISSUE THREAD

**Issue by amacado** [devicons/devicon#377]
The `build icons` action does not create a new branch (and pull request). I assume it's missing a `git add` and `git commit` step? Maybe @Thomas-Boi can have a look at it?

![image](https://user-images.githubusercontent.com/8781699/101551028-a4645c00-39b0-11eb-96bc-a3b044937cfd.png)

**Comment by Thomas-Boi** [comment]
That's strange. According to the Action's documentation, the `branch` parameter is to specify a branch to commit to. Let me double check and see what's happening.

**Comment by Thomas-Boi** [comment]
I found the issue. There's something going wrong with the css generation script:
![image](https://user-images.githubusercontent.com/43018778/101598163-98769b00-39ac-11eb-8c0e-9934ea13c10e.png)

This is interesting considering there is a composer-original font in the repo. I think there might be some issue with the `devicon.json`. I'll look more into it and see what I can find.

**Comment by Thomas-Boi** [comment]
While I was fixing this issue in PR #378,  I found a couple of interesting issues:

1. I found that the create-pr-action would not create a commit if there's no change. This means that both the build icon step and the build css step failed, which was actually what happened.

2. For the build icon step, I found out that the CSS selector I used for Selenium was failing so I have to find new ways to select the element. I think Icomoon might have updated their website and this threw the selector off. 

3. For the build css step, I found out that it was using a nonexistent CSS declaration. This happens because of this section:
![image](https://user-images.githubusercontent.com/43018778/101956603-ba426e80-3bb4-11eb-8d04-98434d2f704c.png)

Long story short, the CSS script assume that whenever a version is put as the base of an alias, it means it's a font. This is not the case for this font where the original version is not made into a font => The script failed. This was done because I assumed we only want aliases for font versions, not svg version.

I have updated the `devicon.json` to fix these issues. I also updated the `CONTRIBUTING.md` to tell future contributors to avoid doing this. We can also consider adding alias to svg versions but I think that'd need an overhaul of the entire system.

The script now runs properly. You can find the PR in the PR menu. I haven't found any other issue so everything is good.

**Comment by Thomas-Boi** [comment]
Regarding the reverted change: I merged it by accident thinking that it has your approval. However, I realize that you said it's not tested so I reverted it. You can still test the PR though so no problem.

**Comment by amacado** [comment]
Your code looks fine, i have no worries to merge it. So I reverted your revert and then released it in the master. Let's see if the build now succeeds *fingers crossed*

**Comment by amacado** [comment]
Looking good! I will manually release a new `gh-pages` branch with the updated icons this weekend so we can see them in the new [release v2.7](https://github.com/devicons/devicon/releases/tag/v2.7).
